app/controllers/v1/users/newsfeed_posts_controller.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class V1::Users::NewsfeedPostsController < ApplicationController
+  before_action :pundit_user
+  before_action do
+    @page = set_page.to_i
+  end
+
+  def index
```

Comment:
Same train of thought as above, you don't need to, it just keeps your controllers clean :)